### PR TITLE
front: fix pluralization in scenario train selection

### LINKED ISSUE #9470: front: pluralization is wrong in scenario train selection
### What happened?

The train word is not pluralized based on the number of trains in scenario train selection

### What did you expect to happen?

_No response_

### How can we reproduce it (as minimally and precisely as possible)?

Create a scenario with a single trainschedule. 
Look at the train selection panel.

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

Unknown

### PR DESCRIPTION
**Pull request by Synar**
Close #9470

### CODE REVIEW

**Review by flomonster — APPROVED** [review]

**Review by theocrsb — APPROVED** [review]
LGTM and tested :)

File: front/public/locales/en/operationalStudies/scenario.json [hunk 1]
```diff
@@ -82,8 +82,11 @@
     "validityFilter": "Trains validity"
   },
   "toggleTimetable": "Toggle the timetable",
-  "trainCount_other": "{{ count }} out of {{ totalCount }} trains selected",
-  "trainCount_zero": "{{ totalCount }} trains",
+  "trainCount_one": "1 out of $t(train, { 'count': {{totalCount}} }) selected",
```
Comment (emersion) [inline front/public/locales/en/operationalStudies/scenario.json:85]: Wouldn't this show "1 out of train selected"?

Comment (emersion) [reply front/public/locales/en/operationalStudies/scenario.json:85]: Ah, no, nevermind!

Comment (emersion) [inline front/public/locales/en/operationalStudies/scenario.json:85]: Nit: do we really need the `trainCount_one` special case? Wouldn't this be taken care of by `trainCount_other`?

Comment (Synar) [reply front/public/locales/en/operationalStudies/scenario.json:85]: This is what I tried first but then something... peculiar happened
[Screencast_20241025_161918.webm](https://github.com/user-attachments/assets/c41dbb06-e29c-4508-b034-1a606ba38379)

Right then and there I decided I would save myself a headache by just being a bit more exhaustive

Comment (emersion) [reply front/public/locales/en/operationalStudies/scenario.json:85]: Oh my.

One less headache sounds good to me!

**Review by emersion — APPROVED** [review]

**Comment by flomonster** [comment]
Also close #9351

**Pull request merged**